wasmtests/names.wat

```diff
@@ -0,0 +1,5 @@
+(module $module_name
+  (func $func_name (local $loc_name i32)
+  )
```

Comment:
small wat module that will have a value in each name subsection when compiled to wasm.